docs/Biblioteca_Aserciones.md

```diff
@@ -0,0 +1,22 @@
+# Biblioteca aserciones
+
+## Criterios
+
+- Mejores prácticas para GO (Bibliotecas referenciadas por la página oficial de Golang).
+- Biblioteca amplia con múltiples opciones.
+- Biblioteca que aporte legibilidad a los tests.
+
+## Opciones
+
+Tras hacer una búsqueda de las opciones que cumplan los criterios se plantean las siguientes opciones, las cuales todas estan referenciadas por Go.
+
+- [BE](https://github.com/carlmjohnson/be?utm_campaign=awesomego&utm_medium=referral&utm_source=awesomego): Es un paquete minimalista de aserciones. Este paquete tiene a su favor que es muy simple y fácil de usar, lo que tambien juega en su contra ya que al ser tan minimalista el abanico de opciones que ofrece es escaso. Se echa en falta funciones que permitan trabajar con errores.

```

Comment:
Al menos podías ver que el URL incluye el referrer de medium, con lo cual está claro de dónde lo has sacado. Edita el URL para que apunte a donde tenga que apuntar.
Por otro lado, ¿qué importa que el abanico de opciones sea escaso? ¿Necesitas esas opciones que *no* ofrece?
Si listáis una biblioteca, enunciad cómo cumple o no los criterios y requisitos. Y punto. Si trabajar con errores es un requisito (debería serlo), enúncialo en los requisitos.